armi/materials/fakeMaterial.py

```diff
@@ -0,0 +1,46 @@
+"""fake test materials for unit tests in armi.reactor.converters.tests.test_axialExpansionChanger"""
+from armi.materials.ht9 import HT9
+from armi.utils import units
+
+
+class Fake(HT9):  # pylint: disable=abstract-method
```

Comment:
Can we change the name of this to "FakeMaterial" or "FakeMat" or something?  

My concern is that a new ARMI user won't know what they are looking at, from the name alone. Thanks!

Reply:
@john-science addressed.

https://github.com/terrapower/armi/pull/606/commits/c9712deea459019976330009bdf19a3bb8b8749d